src/hooks/useConfig.tsx

```diff
@@ -0,0 +1,19 @@
+import { ServerConfig, ServerConfigContainer } from 'containers/ConfigContainer'
```

Comment:
I think this might be duplicating functionality already provided by the *ServerConfigContainer*. See other comment